COMMIT-GUIDELINES.md

```diff
@@ -0,0 +1,161 @@
+1. **Review Before Committing**: Take a moment to review your commit message before finalizing it. Ensure it accurately reflects the changes made.
+
+## Semantic versioning
+
+Semantic versioning is a way of versioning software that follows a structured version numbering convention. It helps identify the significance between versions and is always forward facing, meaning versions are always moving into the future.
+
+The semantic versioning scheme uses three digits to construct the version number. For example, say we had version `5.2.6`:
+
+```
+  5  .  2  .  6
+Major.Minor.Patch
+```
+
+The first digit is the _Major_ version, which is incremented whenever there are breaking changes.
+
+The second digit is the _Minor_ version, which is incremented when there are backwards compatible changes, eg. a new component.
+
+The third digit is the _Patch_ version, which is incremented when there are backwards compatible bug fixes.
+
+### Changelog generation
+
+The commit messages are not only consumed by humans, they are also consumed by the project's tooling when publishing our packages upon release. The project uses [semantic-release](https://github.com/semantic-release/semantic-release) to automatically version the packages, create a changelog for each release, and publish the packages to npm. It does this by tracking the Conventional Commit prefixes on each commit to decide its impact.
+
+The two relevant prefixes are `fix` and `feat`. Commits with these prefixes should have clean messages about what they are fixing or what feature is being added, respectively. Each commit specifies the weight and impact of the commit on the project.
+
+Commits prefixed with `fix` are considered bug fixes, and so they are deemed to have little impact on the project, are backwards compatible with the previous version, and aren't going to cause any breaking changes. These commits trigger a patch version to be generated, which is version increment of `0.0.1` in the semantic versioning scheme.
+
+Commits prefixed with `feat` are considered features, which introduce new functionality into the project, such as a new component, new style change, or other significant change. These commits are backwards compatible as well and trigger a version increment of `0.1.0`.
+
+Not every commit needs to start with `fix` or `feat` within a branch, only the ones that have changes worth calling out in our `CHANGELOG.md` file. Commits can be prefixed with other prefixes: `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`. These are more suited for smaller parts of a larger update or commits that are being made that should not impact the version number, example project documentation updates.
+
+The changelog can be found within the [CHANGELOG.md](https://github.com/ongov/ontario-design-system/blob/develop/CHANGELOG.md) file, which a running history of all changes made in each release. The changelog version entries look like the following example:
```

Comment:
Need to add `is` between `which` and `a`.

e.g. `which is a running history of all changes`